store/transport.go

```diff
@@ -209,3 +214,21 @@ func validateItemTTL(item *model.Item, maxTTL time.Duration) {
 		}
 	}
 }
+
+func isSuperUser(ctx context.Context) bool {
+	auth, ok := bascule.FromContext(ctx)
+	if !ok {
+		return false
+	}
+	attributes := auth.Token.Attributes()
+	attribute, ok := attributes.Get("access-level")
```

Comment:
Oh, I thought you would use the configuration key?

Reply:
Yep, I'll add it back once we have an implementation on reading configs.